tests/behat/settings.feature

```diff
@@ -0,0 +1,42 @@
+@ocis @javascript
+Feature: upload the resource in oCIS to moodle
+  As a user who manages moodle content
+  I want to restrict users access to a drive
+  So that the users can only access what is necessary for them
+
+  Background: 
+    Given I log in as "admin"
+    And I navigate to "Plugins > Repositories > ownCloud Infinite Scale repository" in site administration
+    And I click on "Settings" "link"
+
+
+  Scenario Outline: hide personal/shares drive
+    When I change the visibility of "<drive>" drive to "No"
+    And I follow "Profile" in the user menu
+    And I click on "Blog entries" "link"
+    And I click on "Add a new entry" "link"
+    And I click on "Add.." "button"
+    And I click on "oCIS" "link"
+    And I click on "Log in to your account" "button"
+    And I switch to a second window
+    And I log in to ocis as "admin"
+    Then I should not see "<drive>"
```

Comment:
please check if the other drives are still visible, only having a negative check is dangerous. e.g. if something in the UI changes and the element cannot be found the test will always pass.

You could make a step: `Then I should only see ....`